trading/exchanges/exchange_manager.py

```diff
@@ -172,6 +172,9 @@ def _set_config_time_frame(self):
         if client_shortest_time_frame not in self.time_frames:
             self.time_frames.append(client_shortest_time_frame)
 
+        self.time_frames = TimeFrameManager.sort_time_frames(self.time_frames)
+        self.time_frames.reverse()
```

Comment:
I think we can simplify here with the [`reverse=True` ](https://www.programiz.com/python-programming/methods/built-in/sorted) arg of `sorted()`.

Reply:
Sure, was not implemented in TimeFrameManager.sort_time_frames, I added it, better like this 👍